administrator/language/en-GB/en-GB.com_media.ini

```diff
@@ -59,6 +60,7 @@ COM_MEDIA_MEDIA_NAME="Name"
 COM_MEDIA_MEDIA_SIZE="Size"
 COM_MEDIA_MEDIA_TYPE="Type"
 COM_MEDIA_NAME="Name"
+COM_MEDIA_NEW_FOLDER="New folder"
```

Comment:
Based on your screenshot I assume this should be New Folder

Reply:
I think, I not updated the screenshot. It's fine it should be small right ?

Reply:
It should be as in the screenshot